src/frontend/js/components/AshleyEditor/LatexButton/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useRef } from 'react';
+import { DraftJsButtonTheme } from '@draft-js-plugins/buttons';
+import { EditorState } from 'draft-js';
+import {
+  insertInlineTeX,
+  insertTeXBlock,
+} from 'draft-js-latex-plugin/lib/utils/insertTeX';
+import { TypeLatexStyle } from '../../../types/Enums';
+interface LatexButtonProps {
+  type: string;
+  theme: DraftJsButtonTheme;
+  getEditorState: () => EditorState;
+  setEditorState: (editorState: EditorState) => void;
+}
+
+export const LatexButton = (props: LatexButtonProps) => {
+  const node = useRef<HTMLDivElement>(null);
+  const { buttonWrapper = '', button } = props.theme;
+  const imageClassName =
+    props.type === TypeLatexStyle.INLINE ? 'latex-inline' : 'latex-block';
```

Comment:
Easier to maintain ?
```suggestion
  const imageClassName = useMemo(() => `latex-${props.type}`, [props.type]);
```